docs/source/gateway.md

```diff
@@ -45,87 +39,53 @@ Hyperledger Fabric currently supports client application development in three la
+The Fabric Gateway manages the complexity of transaction endorsement on behalf of the client, using the following process:
+
+- The Fabric Gateway selects the endorsing peer from the gateway peer's organization (MSPID) by identifying the (available) peer with the highest ledger block height. The assumption is that all peers within the gateway peer's organization are *trusted* by the client application that connects to the gateway peer.
+- The transaction proposal is simulated on the selected endorsement peer. This simulation captures information about the accessed states, and therefore the endorsement policies to be combined (including any individual state-based policies
+stored on the endorsement peer's ledger).  
+- The captured policy information is assembled into a `ChaincodeInterest` protobuf structure and passed to the discovery service in order to derive an endorsement plan specific to the proposed transaction.
+- The gateway applies the endorsement plan by requesting endorsement from the organizations required to satisfy all policies in the plan. For each organization, the gateway peer requests endorsement from the (available) peer with the highest block height.
+
+The gateway is dependent on the [discovery service](discovery-overview.html) to get the connection details of both the available peers and ordering service nodes, and for calculating the combination of peers that are required to endorse the transaction proposal. The discovery service must therefore always remain enabled on the Fabric Gateway peer.
+
+The gateway endorsement process is more restrictive for transient data because it often contains sensitive or personal information that must not persist in the ledger. In this case, the gateway will restrict the set of endorsing organizations to those that are members of the private data collection to be accessed (either read or write). If this restriction for transient data results in failed endorsement, the gateway rejects the transaction proposal. If rejected, the client should [explicitly define which organizations should endorse](#how-to-override-endorsement-policy-detection) the transaction, which must be restricted to members of the private data collection.
+
+### Targeting specific endorsement peers
+
+In some cases, a client application must explicitly identify the specific organizations to evaluate or endorse a transaction proposal. Transient data, for example, often contains personal or sensitive information that must only be written to a shared private data collection. In these cases, the client application can explicitly specify the endorsing organizations; the gateway will select the available peer with the highest block count from each organization. However, if the inclusion of an organization violates a required endorsement policy, then the transaction is invalidated by all of the requested endorsing peers. This failed endorsement is recorded on the ledger but no transaction writes will exist in the state database.
```

Comment:
"However, if the inclusion of an organization violates a required endorsement policy, then the transaction is invalidated by all of the requested endorsing peers."

"inclusion of an organization" is awkward here. It would be better to say something like "if the specified endorsing organizations violates the endorsement policy"...

Also, "then the transaction is invalidated by all of the requested endorsing peers" is not correct. The endorsing peers will successfully return a proposal response. Once the transaction gets included in a block, EVERY peer on the channel (not just the endorsing peers) will invalidate the transaction and not update its state database.

Reply:
So there's this concept of a "plan" when there are multiple endorsement policies that must pass. I think just referencing multiple policies is easier to digest than an additional concept. Here you say "the endorsement policy" as if there is only one policy, please confirm.

Reply:
And what is the proposal response in this case. failed, e.g.

Reply:
So we currently have this: 
`In some cases, a client application must explicitly select the organizations to evaluate or endorse a transaction proposal. For example, transient data often contains personal or sensitive information which must only be written to a  private data collection, thereby limiting the pool of endorsement organizations. In these cases, the client application can explicitly specify the endorsing organizations; the gateway will then select the (available) endorsing peer with the highest block count from each organization. However, if the selection of one or more endorsing organizations violates an endorsement policy, then the transaction is invalidated by all peers in the channel. This failed endorsement is recorded on the ledger but the transaction is not written to the state database on any channel peer.`

Reply:
You're right, one transaction may have multiple endorsement policies due to chaincode-level, collection-level, and state-based endorsement policies. All of these have to be satisfied for the transaction to be validated. Sometimes we refer to this whole set as the endorsement policy of the transaction as a whole.

Reply:
> And what is the proposal response in this case of failure for one or more of the endorsement policies. failed, e.g.

Remember, endorsement policy is not checked until validation/commit time later. The proposal and proposal response are ignorant of the endorsement policy. The gateway peer however does inspect the first proposal response and work with discovery service to identify the endorsement policy of the transaction and route the proposal to the required organization's peers.